### PR TITLE
handle errors when updating module version

### PR DESCRIPTION
**Pull request by mirpedrol**
Handle errors when trying to update a module version.
Needed for [#1745](https://github.com/nf-core/modules/pull/1745)

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by ErikDanielsson — APPROVED** [review]
Looks good! Just have look at my comment for a minor suggestion (:

File: nf_core/modules/lint/main_nf.py [hunk 1]
```diff
@@ -281,24 +281,31 @@ def check_process_section(self, lines, fix_version, progress_bar):
...
-                        self.passed.append(
-                            (
-                                "bioconda_latest",
-                                f"Conda package has been updated to the latest available: `{bp}`",
-                                self.main_nf,
-                            )
-                        )
+                    try:
+                        fixed = _fix_module_version(self, bioconda_version, last_ver, singularity_tag, response)
+                    except Exception as e:
```
[the remark this answers is not in the document]
Comment (mirpedrol) [reply nf_core/modules/lint/main_nf.py:286]: Thanks for the suggestion! 
I will wait to merge this until I have the GHA working. I assume I will find more errors while testing it, so will be able to know which specific types we should handle.

**Comment by mirpedrol** [comment]
I am going to merge this, it finally handles the annoying error with sqlite :) It's something related to a timeout, it's not failing locally.
Some things have changed since the review, if you want to have a final look @ErikDanielsson

**Comment by ErikDanielsson** [comment]
Code still looks good to me! But you should update the changelog :)

**Comment by mirpedrol** [comment]
thanks!

**Pull request merged**